kickstarter/android-oss
# EP - 272: Login Initiate Cta Click

# 📲 What

- Register login initiate CTA clicked event to segment

# 🤔 Why

Segment Integration

# 🛠 How

- Create new method ``` trackLogInInitiateCtaClicked ``` in ```AnalyticEvents ``` class
- Called method from ``` LoginToutViewModel ``` on ``` login with email ``` button clicked

# 👀 See

<img width="363" alt="Screenshot 2021-03-11 at 17 22 43" src="https://user-images.githubusercontent.com/63934292/111017916-6ebb7e80-83b6-11eb-8e39-3323b96ef041.png">


| Before 🐛 | After 🦋 |
| --- | --- |
|  |  |

# 📋 QA

- Navigate to ``` Login Or Sign Up ``` Screen.
- Click on ``` Login with Email ```
- From the segment console, you should see the new event registered

<img width="732" alt="Screenshot 2021-03-13 at 04 40 51" src="https://user-images.githubusercontent.com/63934292/111017959-afb39300-83b6-11eb-857f-57e09346ee80.png">


# Story 📖

https://kickstarter.atlassian.net/browse/EP-272
